File: src/main/java/com/likelionknu/applyserver/admin/data/dto/request/ApplicationStatusUpdateRequestDto.java
package com.likelionknu.applyserver.admin.data.dto.request;

import com.likelionknu.applyserver.auth.data.enums.ApplicationEvaluation;
import com.likelionknu.applyserver.auth.data.enums.ApplicationStatus;
import jakarta.validation.constraints.NotNull;

public record ApplicationStatusUpdateRequestDto(
        ApplicationStatus status,
        ApplicationEvaluation evaluation
) {}

File: src/main/kotlin/com/likelionknu/applyserver/admin/data/dto/request/ApplicationStatusUpdateRequestDto.kt
package com.likelionknu.applyserver.admin.data.dto.request

import com.likelionknu.applyserver.auth.data.enums.ApplicationEvaluation
import com.likelionknu.applyserver.auth.data.enums.ApplicationStatus

data class ApplicationStatusUpdateRequestDto(
    val status: ApplicationStatus?,
    val evaluation: ApplicationEvaluation?
)